### PR TITLE
Add ISIS Reflectometry GUI Preview Tab manual testing instructions

### LINKED ISSUE #34967: Add manual testing documentation for the ISIS Reflectometry GUI preview tab
We used the testing instructions [here](https://github.com/mantidproject/mantid/issues/31069) for manually testing the preview tab for the 6.6 release, but we need to update these and add them to the online manual testing documentation.

During [testing for the 6.6 release](https://github.com/mantidproject/mantid/issues/34952) we felt that the `INTER45455_inst` test data made it difficult to tell if moving the ROIs on the spectrum viewer was updating the plot (the reduction was clearly being re-run, but we concluded that the data changes must be so small that you couldn't see the plot changing). If possible, it would be good to find some test data that made this clear.

### PR DESCRIPTION
**Pull request by rbauststfc**
**Description of work.**

Adds manual testing instructions for the new preview tab in the ISIS Reflectometry GUI. We're using test data from the unit tests at the moment - the reason for this is explained on the linked issue and it shouldn't be necessary long-term.

**To test:**

Check that the new instructions make sense.

Fixes #34967. 

*This does not require release notes* because **it is an update to documentation.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by cailafinn — CHANGES_REQUESTED** [review]
Reads well and tests almost all the functionality I can think of, just a few suggestions:

File: dev-docs/source/Testing/ReflectometryGUI/ReflectometryGUITests.rst [hunk 1]
```diff
@@ -169,3 +170,30 @@ Export as ASCII
 - Change the dropdown to ``ILL Cosmos (*.mft)``. The settings remain greyed out apart from the parameters which should now be enabled. Click save and you should get 3 columns of numbers padded by spaces (including leading spaces). There should be a header starting ``MFT`` which includes any parameters you selected, the same as the Custom format.
 
 - Try entering a non-existent or invalid save path and then try to Save. You should get an error saying that the path is invalid.
+
+Preview tab
+---------------
+
+- Go to the Reduction Preview tab.
+- Type `INTER45455` into the `Run` input. Set the `Angle` to `1` and click `Load`. The instrument view plot should display the data. If the plot remains blank then check that you have added the path to your unit test data to your Mantid user directories (see set up instructions).
```
[the remark this answers is not in the document]
Comment (rbauststfc) [reply dev-docs/source/Testing/ReflectometryGUI/ReflectometryGUITests.rst:178]: Ah, thanks, I'll correct that - I'm not sure why I didn't spot it from the existing text!

[the remark this answers is not in the document]
Comment (rbauststfc) [reply dev-docs/source/Testing/ReflectometryGUI/ReflectometryGUITests.rst:178]: It's been re-named in the unit test data, so this should work OK.

File: dev-docs/source/Testing/ReflectometryGUI/ReflectometryGUITests.rst [hunk 2]
```diff
@@ -169,3 +170,30 @@ Export as ASCII
...
+- Click the rectangle-select button above the plot and draw a region onto the instrument view. The selected detector segments should be summed and the result plotted on the second (slice viewer) plot.
```
[the remark this answers is not in the document]
Comment (rbauststfc) [reply dev-docs/source/Testing/ReflectometryGUI/ReflectometryGUITests.rst:179]: Yes, good idea. I've just checked the deleting and I found that I could use the delete key for both the instrument viewer and slice viewer. I had to be quite careful that the region was definitely selected, though. Are you still having this problem?

[the remark this answers is not in the document]
Comment (ConorMFinn) [reply dev-docs/source/Testing/ReflectometryGUI/ReflectometryGUITests.rst:179]: https://github.com/mantidproject/mantid/pull/35131/files/02c0666414d92f6ee5926f44bd8801eccc825531..ceeab53a9a73f9c4a93b9a2db4907f0b6021bcda#r1102894614

[the remark this answers is not in the document]
Comment (rbauststfc) [reply dev-docs/source/Testing/ReflectometryGUI/ReflectometryGUITests.rst:179]: Ah, thanks for double checking. I'll add it to the list on issue #34220 as something we need to look into.

**Review by cailafinn — APPROVED** [review]
Looks good to me, good to get these added in 👍

**Pull request merged**